docs/docs/legal/service-level-description.md

```diff
@@ -0,0 +1,80 @@
+
+## Definitions
+
+**Monthly Uptime Percentage** means total number of minutes in a month, minus the number of minutes of Downtime suffered from all Downtime Periods in a month, divided by the total number of minutes in a month.
+
+**Downtime Period** means a period of one or more consecutive minutes of Downtime. Partial minutes or intermittent Downtime for a period of less than one minute will not count towards any Downtime Period.
+
+**Downtime** means any period of time in which Core Services are not Available within the Region of the customer’s organization. Downtime excludes any time in which ZITADEL Cloud is not Available because of
+- Announced maintenance work
+- Emergency maintenance
+- Force majeure events.
+
+**Available** means that Core Services of ZITADEL Cloud respond to Customer Requests in such a way that results in a Successful Minute. The Availability of Core Services will be monitored from CAOS’ facilities from black-box monitoring jobs.
+
+**Successful Minute** means a minute in which ZITADEL cloud is not repeatedly returning Failed Customer Requests and includes minutes in which no Customer Request were made.
+
+**Customer Requests** means a HTTP GET request made by a Customer or a Customers’ users to Core Services within the Customer’s organization’s region.
+
+**Successful Minute** means a minute in which Auth0 is not repeatedly returning Failed Customer Requests and includes minutes in which no Customer Requests were made.
+
+Failed Customer Request means Customer Requests that 
+- Returns an server error; or
+- is received by ZITADEL Cloud and results in no response where one is expected
+
+This excludes specifically: 
+- Failed Customer Requests due to malformed requests, client-side application errors outside of ZITADEL Cloud’s control
+- Customer Requests that do not reach ZITADEL Cloud Core Services
+
+**Core Services** means the following ZITADEL Cloud Services and API’s: 
+- **Authentication API** Endpoints
+- **OpenID Connect 1.0 / OAuth 2.0 API** Endpoints
+- **Login Service** means the graphical user interface of ZITADEL Cloud for users to Login, Self-Register, and conduct a Password Reset.
+- **Identity Brokering Service** means the component of ZITADEL Cloud that handles federated authentication of users with third-party identity provider, excluding any failure or misconfiguration by the third-party
+
+**Financial Credit** means the percent of the monthly subscription fee applicable to the month in which the guaranteed service level was not met, according to the actual achieved monthly uptime percentage, as shown in the following table
+
+Achieved vs.  Guaranteed| 99.50% | 99.90% | 99.95%
+--- | --- | --- | ---
+99.5% - < 99.9% | n/a | n/a | 10%
+99.0% - < 99.5% | n/a | 10% | 25%
+95.0% - < 99.0% | 10% | 25% | 50%
+< 95.0% | 50% | 50% | 50%
+
+
+## Service Levels
+
+### Availability Objective
```

Comment:
Should we also state here that no further damages can be claimed when we are offline?

Reply:
"Service Credits shall be the sole and exclusive remedy for breach of this SLA."?